src/AutosuggestContainer.js

```diff
@@ -46,8 +46,8 @@ function mapToAutowhateverTheme(theme) {
   return result;
 }
 
-export default class AutosuggestContainer extends Component {
-  static propTypes = {
+export default React.createClass({
```

Comment:
I'd probably keep this as a class since the React ["eventual goal is for ES6 classes to replace React.createClass completely"](https://facebook.github.io/react/blog/2015/12/18/react-components-elements-and-instances.html).